### DISCUSSION TITLE
Skeleton without video

### DISCUSSION THREAD

**Discussion by TheMalaland** [talmolab/sleap#921] [Ideas]
Is there a way of getting a video with only the skeleton movements, i mean black screen  with overlapping skeleton  predictions . I think this feature could be something useful for visual movement analisys.

**Comment by roomrys** [comment]
Hi @TheMalaland,

We do not have a feature in the SLEAP GUI to do this yet; however, if you have the source code downloaded, then a simple change to this function:
https://github.com/talmolab/sleap/blob/2688d56172eb297a25d33a47a0b4bdc84a5652d9/sleap/io/video.py#L1078-L1088
with the `return` multiplied by 0 will return a black frame.

You can then View > Render Video Clip with Instances... to generate a black video with instances overlaid. 

This would be a great first feature to contribute to SLEAP. 

Thanks for the idea!
Liezl

**Comment by auesro** [comment]
It would be even better if you can choose the color of the background image: black, white, grey.

Thanks!

A

**Reply by roomrys** [reply]
Long, awaited, but finally integrated. This has been included in the SLEAP 1.4.1 release!

**Comment by juliagorman** [comment]
It would also be cool if this could also work for the 3D version too
